.github/compatibility-test-matrices/release-script.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Get the directory path where JSON files are located
+directory_path=$(dirname "$0")
+
+# Create a backup directory with the current date and time
+backup_dir="${directory_path}_backup_$(date +%Y-%m-%d-%H-%M-%S)"
+mkdir "$backup_dir"
+
+# Copy the contents of the directory to the backup directory
+cp -R "$directory_path"/* "$backup_dir"
+
+# Step 1: Replace the release version in JSON files
+echo "Enter the release version to replace (ie: v4.4.0, leave empty to skip):"
+read old_release_version
```

Comment:
Looks like this script is doing two things, right? Do you think it warrants being split up?

If keeping as a single script then a general UX remark could be to present a menu and select (1) or (2) to choose which action to perform rather than a CR input to skip here.

Reply:
yea, it was the two possible options that i saw carlos doing with release versions (either replacing due to security) or adding a new one. I can split it up, good idea